src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -188,6 +192,15 @@ private function storeObjectToPublish($object, string $property): void
             if (!isset(self::ALLOWED_KEYS[$key])) {
                 throw new InvalidArgumentException(sprintf('The option "%s" set in the "mercure" attribute of the "%s" resource does not exist. Existing options: "%s"', $key, $resourceClass, implode('", "', self::ALLOWED_KEYS)));
             }
+
+            if ('hub' === $key && !$this->registry instanceof HubRegistry) {
+                throw new InvalidArgumentException(sprintf('The option "hub" set in the "mercure" attribute of the "%s" resource is not supported. Please install symfony/mercure:^0.5 to enable it.', $resourceClass));
+            }
+        }
+
+        $options['enable_async_update'] = $options['enable_async_update'] ?? true;
+        if ($options['enable_async_update']) {
+            @trigger_error(sprintf('Not setting "enable_async_update" option set in the "mercure" attribute of the "%s" resource to "false" is deprecated.', $resourceClass), \E_USER_DEPRECATED);
```

Comment:
We should only deprecate this in main, we try to not introduce deprecations in patch release except for security issues.

Reply:
May we add a TODO then?

Reply:
And the same goes for other deprecations I  can guess

Reply:
I didn't add a todo, I'll create a PR aiming to add these deprecations instead